src/DependencyInjection/SonataPageExtension.php

```diff
@@ -17,15 +17,26 @@
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\DependencyInjection\Reference;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class SonataPageExtension extends Extension
+class SonataPageExtension extends Extension implements PrependExtensionInterface
 {
+    public function prepend(ContainerBuilder $container)
+    {
+        if ($container->hasExtension('twig')) {
+            // add custom form widgets
+            $container->prependExtensionConfig('twig', [
+                'form_themes' => ['@SonataCore/Form/datepicker.html.twig'],
```

Comment:
Will this append or replace the current `form_themes` config?

Reply:
This will prepend ) Application can have a lot of configuration files for a bundle, container will merge these files into one. This array is something like a new file, but its content is placed to the begining of the list.

I found `prepend` method in [symfony-cmf/core-bundle](https://github.com/symfony-cmf/core-bundle/blob/master/src/DependencyInjection/CmfCoreExtension.php) and in [sonata-project/user-bundle](https://github.com/sonata-project/SonataUserBundle/blob/4.x/src/DependencyInjection/SonataUserExtension.php). A bundle can configure another bundle =) But this configuration can be overriden with real config files